kahuna/public/js/search/results.html

```diff
@@ -69,22 +69,25 @@
                 </ul>
             </div>
 
-            <div class="results-toolbar__right flex-container" ng:if="ctrl.inSelectionMode">
-                <div class="results-toolbar-item results-toolbar-item--right results-toolbar-item--static">
+            <div class="results-toolbar__right flex-container">
+                <div class="results-toolbar-item results-toolbar-item--right results-toolbar-item--static"
+                     ng:if="ctrl.selectionCount > 0">
```

Comment:
I didn't think it was a massive problem as it keeps separated...  Happy to move it into a `selection-actions` component if you think that is more clear.